src/components/message.tsx

```
@@ -56,22 +63,96 @@
+    (
+      | undefined
+      | NonNullable<
+          FunctionReturnType<typeof api.chats.getChatInfoFromId>
+        >["otherUser"]
+    ),
+  ];
+}) => {
+  // Group reactions by emoji
+  const reactionsByEmoji = reactions.reduce(
+    (acc, reaction) => {
+      (acc[reaction.emoji] = acc[reaction.emoji] ?? []).push(reaction);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)